examples/pipeline_wavernn/README.md

```diff
@@ -0,0 +1,36 @@
+This is an example pipeline for WaveRNN vocoder using the WaveRNN model trained with LJSpeech. WaveRNN model is based on the implementation from https://github.com/fatchord/WaveRNN. The original implementation was
```

Comment:
nit: from [this repository](https://github.com/fatchord/WaveRNN).

Reply:
updated.